### PR TITLE
Added TLS documentation [4648].

### PR DESCRIPTION
**Pull request by LuisGP**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/37/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/43/)

File: docs/advanced.rst [hunk 1]
```diff
@@ -230,6 +230,51 @@ example.
 
 TCP doesn't support multicast scenarios, so you must plan carefully your network architecture.
 
+
+Using TLS (Transport Layer Security) over TCP
+---------------------------------------------
+
+Fast-RTPS allows configuring a TCP Transport to use TLS. We must configure both **TCP Server** and **TCP Client**
```
Comment (richiware) [inline docs/advanced.rst]: Break the line. And second line is a little repetition. Also `We must set TCP server and TCP Client propertly`

File: docs/xmlprofiles.rst [hunk 2]
```diff
@@ -155,13 +155,82 @@ The XML label ``<transport_descriptors>`` can hold any number of ``<transport_de
...
 RTCP is the control protocol for communications with RTPS over TCP/IP connections.
 
 There are more examples of transports descriptors in :ref:`comm-transports-configuration`.
 
+.. _tcp-tls:
+
+TLS Configuration
+^^^^^^^^^^^^^^^^^
+
+Fast-RTPS allows configure TLS parameters through the ``<tls>`` tag of its Transport Descriptor.
```
Comment (richiware) [inline docs/xmlprofiles.rst]: `to configure` or `configuring`

File: docs/xmlprofiles.rst [hunk 3]
```diff
@@ -155,13 +155,82 @@ The XML label ``<transport_descriptors>`` can hold any number of ``<transport_de
 |                               | work as TCP client only           |                                 |                |
 |                               | (TCP **only**).                   |                                 |                |
 +-------------------------------+-----------------------------------+---------------------------------+----------------+
+| ``<tls>``                     | Allow to define TLS related       | :ref:`tcp-tls`                  |                |
```
Comment (richiware) [inline docs/xmlprofiles.rst]: `Allows`

File: docs/advanced.rst (hunk 1 above)
Comment (LuisGP) [reply docs/advanced.rst]: What about "Fast-RTPS allows configuring a TCP Transport to use TLS by setting up **TCP Server** and **TCP Client** properly."?

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/44/)

**Pull request merged**